src/main/java/no/unit/nva/database/DatabaseIndexDetails.java

```diff
@@ -0,0 +1,11 @@
+package no.unit.nva.database;
+
+public final class DatabaseIndexDetails {
+
+    public static final String SECONDARY_INDEX_1_HASH_KEY = "SecondaryIndex1HashKey";
```

Comment:
index names and key names cannot be loaded in runtime. They should be available in compilation time. Collect them in one class for better overview